target/linux/ar71xx/config-4.4

```diff
@@ -121,6 +121,7 @@ CONFIG_ATH79_MACH_GL_MIFI=y
 CONFIG_ATH79_MACH_GL_USB150=y
 CONFIG_ATH79_MACH_GS_MINIBOX_V1=y
 CONFIG_ATH79_MACH_GS_OOLITE=y
+CONFIG_ATH79_MACH_DPT_MODULE_V1=y
```

Comment:
this symbol also needs to be added to the mikrotik and nand config as "is not set"